### PR TITLE
set up styled components

### LINKED ISSUE #17: Setting up CSS and styling

### PR DESCRIPTION
**Pull request by hanlinc27**
* added styled components package
* added an example button + example colour change 
* did _not_ add stylelint --> do we want to use style lint or ESlint?

### CODE REVIEW

**Review by lesliexin — COMMENTED** [review]
looks good to me :)

**Review by lesliexin — APPROVED** [review]

File: ui/src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,35 @@
 import * as React from 'react';
+import styled from "styled-components";
```
Comment (AbhiPrasad) [inline ui/src/App.tsx:2]: Can we make sure to `yarn add styled-components` and update the `package.json`?

**Review by AbhiPrasad — CHANGES_REQUESTED** [review]
Quick note about updating `package.json`.

Can we also install the babel plugin? Instructions here: https://styled-components.com/docs/tooling#babel-plugin.

**Review by dhillondeep — APPROVED** [review]

**Review by dhillondeep — APPROVED** [review]

**Comment by dhillondeep** [comment]
@AbhiPrasad requested changes review!

**Review by AbhiPrasad — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**